marlowe-integration/marlowe-integration.cabal

```diff
@@ -55,18 +55,16 @@ library
     , aeson
     , async
     , async-components
-    , base >=4.9 && <5
+    , base                                                                                                                                          >=4.9 && <5
     , bytestring
     , cardano-api
     , cardano-crypto-wrapper
     , cardano-integration
     , cardano-ledger-byron
-    , co-log >=0.5.0.0 && <0.6.0.0
     , containers
-    , eventuo11y >=0.9 && <0.11
+    , eventuo11y                                                                                                                                    >=0.9 && <0.11
```

Comment:
`cabal-fmt --inplace --no-tabular **/*.cabal && git commit -am "format cabal files" --no-verify` maybe you want to add this extra commit at the end ? 
@jhbertra what do you think ?

Reply:
If you rebase and run `pre-commit run --all` this should be reverted now.

Reply:
although `hlint` will fail until #626 is merged.

Reply:
Fixed in 34f1d7e5d89e68996254f165386a3264372c3252.